client/src/app/site/pages/meetings/pages/motions/services/list/motion-list-sort.service/motion-list-sort.service.ts

```diff
@@ -80,11 +85,19 @@ export class MotionListSortService extends BaseSortListService<ViewMotion> {
     protected async getDefaultDefinition(): Promise<OsSortingDefinition<ViewMotion>> {
         await this.defaultSortingLoaded;
         return {
-            sortProperty: this.defaultMotionSorting as keyof ViewMotion,
+            sortProperty: this.getDefaultSortProperty(),
             sortAscending: true
         };
     }
 
+    private getDefaultSortProperty(): OsSortProperty<ViewMotion> {
+        if (this.defaultMotionSorting === `weight`) {
+            return [`tree_weight`, `id`];
```

Comment:
The `tree_weight` should always be unique I think, is the second sorting condition really necessary?

Reply:
Idk, I just applied the changes to the sort option properties from PR #1475 to the default properties, I thought maybe the `tree_weight` might not be set in certain conditions, I honestly found it stranger that the secondary sorting condition was `id` and not `sequential_number` or `number`.

Btw if it is only `tree_weight`, we may as well change the backend option `weight` to `tree_weight` for the `motions_default_sorting` setting, this would require the client settings to be changed, but at least it wouldn't require this extra if condition in the long term. What do you think @jsangmeister ?

Reply:
Ah, I see - please check if it works without the `id` and change it here and above if it does. I don't know about changing the default in the backend - I see the value, but at the same time `tree_weight` is rather client-specific and might confuse someone reading the models file, so I'm on the fence about it.